packages/components/card/src/Card.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <div class="ix-card" :class="classes">
+    <div class="ix-card-head" v-if="isShowTitle || isShowExtra">
+      <div v-if="isShowTitle" class="ix-card-head-title">
+        <slot name="title">
+          <span>{{ title }}</span>
+        </slot>
+      </div>
+      <div v-if="isShowExtra" class="ix-card-head-extra">
+        <slot name="extra">
+          <span>{{ extra }}</span>
+        </slot>
+      </div>
+    </div>
+    <ix-spin :spinning="isShowLoading" tip="加载中...">
```

Comment:
```suggestion
    <ix-spin :spinning="isShowLoading">
```

不给 tip 吧, 如果要给 tip 也不能直接写死，要用 i18n